### PR TITLE
web-ui: Add success message for no violations

### PR DESCRIPTION
**Pull request by bhaskarmelkani**
Add success message for no violations.
Related to #197

### CODE REVIEW

File: web-ui/src/client/app/components/complete-msg.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  constructor(props) { super(props) }
+
+  render(){
+    if( this.props.error ){
+
+      return <Msg type="error" title="ERROR" text={this.props.error} onCloseButtonClick={this.props.onCloseError} />;
+
+    }else if( this.props.ajaxComplete && !this.props.violations.length ){
+
+      return <Msg type="success" title="Good Job !" text="The swagger file does not have any violations" onCloseButtonClick={this.props.onCloseSuccess} />
```
Comment (fmueller) [inline web-ui/src/client/app/components/complete-msg.jsx]: Can you make the title and text configurable? :-)

Comment (bhaskarmelkani) [inline web-ui/src/client/app/components/complete-msg.jsx]: Yeah sure, and I guess its better to get it from API.
Right now there is not message for success in the API.

**Comment by bhaskarmelkani** [comment]
@rbarilani Yeah updated the code.

File: web-ui/src/client/app/components/form.jsx [hunk 2]
```diff
@@ -26,19 +26,24 @@ export default class Form extends React.Component {
   }
 
   clearError() {
-    this.setState({ error: null });
+    this.setState({ error: null, ajaxComplete: false });
+  }
+
+  resetAjaxComplete() {
```
Comment (rbarilani) [inline web-ui/src/client/app/components/form.jsx]: @bhaskarmelkani Where do you use this method? If not used remove it!

File: web-ui/src/client/app/components/violations.jsx [hunk 3]
```diff
@@ -62,4 +63,23 @@ function ViolationPaths(props) {
   )
 }
 
-export {Violations, Violation, ViolationPaths, ViolationRuleLink, ViolationType}
+function ViolationsResult(props){
+  return (
+    <div className="violations-container">
```
Comment (rbarilani) [inline web-ui/src/client/app/components/violations.jsx]: @bhaskarmelkani rename the classNAme violations-container as violations-result

File: web-ui/src/client/app/components/violations.jsx [hunk 4]
```diff
@@ -62,4 +63,23 @@ function ViolationPaths(props) {
...
+      <If test={() => props.pending}>
+        <div className="violations-container__spinner"><div className="dc-spinner dc-spinner--small"></div></div>
```
Comment (rbarilani) [inline web-ui/src/client/app/components/violations.jsx]: @bhaskarmelkani rename violations-container__spinner as violations-result__spinner

File: web-ui/src/client/app/components/form.jsx [hunk 5]
```diff
@@ -1,6 +1,5 @@
 import React from 'react';
-import {Violations} from './violations.jsx'
-import {Msg} from './dress-code.jsx';
+import {Violations, ViolationsResult} from './violations.jsx'
 
 const css = `
     .violations-container__spinner {
```
Comment (rbarilani) [inline web-ui/src/client/app/components/form.jsx]: @bhaskarmelkani After renaming the class... the css style must be renamed accordingly...: violations-result__spinner

Comment (bhaskarmelkani) [inline web-ui/src/client/app/components/form.jsx]: Done

File: web-ui/src/client/app/components/form.jsx (hunk 2 above)
Comment (bhaskarmelkani) [inline web-ui/src/client/app/components/form.jsx]: Not used after changing the implementation, removed

File: web-ui/src/client/app/components/violations.jsx (hunk 3 above)
Comment (bhaskarmelkani) [inline web-ui/src/client/app/components/violations.jsx]: Done

**Comment by bhaskarmelkani** [comment]
👍

**Comment by rbarilani** [comment]
👍

**Comment by netme** [comment]
@rbarilani should I merge it?

**Comment by rbarilani** [comment]
@netme Yep.

**Comment by netme** [comment]
👍

**Comment by netme** [comment]
Waiting for Travis to pass

**Pull request merged**